### PR TITLE
Culturology series

### PR DESCRIPTION
**Pull request by laurenwigmore**
blog for series & ep 2-3-4 podcast page

### CODE REVIEW

File: content/blog/2023/08/30/cultural-evolution-a-new-discipline-is-born-series.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Episode 1
+
+ <iframe width="560" height="315" src="https://www.youtube.com/embed/vYzOr7l-U20?si=apJFeFtCREo0PZ-2" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### Episode 2 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/gS_lpJnlCh0?si=C5WIhL7_2MVhQnCh" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Episode 3
```
Comment (nathenf) [inline content/blog/2023/08/30/cultural-evolution-a-new-discipline-is-born-series.md:33]: This one needs to be turned into a heading too

File: content/blog/2023/08/30/cultural-evolution-a-new-discipline-is-born-series.md
Comment (nathenf) [inline content/blog/2023/08/30/cultural-evolution-a-new-discipline-is-born-series.md:1]: I would almost make this the Learn page and make the blog more simple like "New Podcast Series Launched: Cultural Evolution with Prof. Joseph Henrich and Dr. Rufus Pollock"

File: content/podcast/Cultural-Evolution-Culture-Cultural-Evolution-and-the-Scaling-of-Societies.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+date: 2023-09-09
+src: https://anchor.fm/s/????.m4a
+---
+In Episode 3 of the series Joseph Henrich and Rufus Pollock continue on the Culture Evolution conversation.
+
+In this episode the significance of family structures and the church takes center stage in their role towards shaping human societies. Joseph explains that different kinship networks influence behaviors, trust, and cooperation within societies and how the Catholic Church played an unintentionally role in shaping Western societies by implementing rules against cousin marriage and polygyny. These rules inadvertently fostered individualism, trust in non-kin relationships, and analytic thinking. These cultural shifts and networks of horizontal connections led to the development of "WEIRD" psychology – Western, Educated, Industrialized, Rich, and Democratic, a topic discussed in Henrich’s latest book. These cultural transformations resulted in the development of different psychological traits that help to explain the remarkable economic success, innovations, and current challenges faced by Western societies.
+
+This conversation forms part of the Cultural Evolution: A New Discipline is Born Series.
```
Comment (nathenf) [inline content/podcast/Cultural-Evolution-Culture-Cultural-Evolution-and-the-Scaling-of-Societies.md:11]: Link this to the learn page once created

File: content/podcast/Cultural-Evolution-Culture-Cultural-Evolution-and-the-Scaling-of-Societies.md
Comment (nathenf) [inline content/podcast/Cultural-Evolution-Culture-Cultural-Evolution-and-the-Scaling-of-Societies.md:1]: Just a note on the URL here we don't want to use CAPs in any URL

File: content/podcast/Cultural-Evolution-Implications-and-whats-next.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
+---
+title: "Joseph Henrich and Cultural Evolution: Implications and what's next"
```
Comment (nathenf) [inline content/podcast/Cultural-Evolution-Implications-and-whats-next.md:2]: Inconsistency in using CAPs in the titles - see the last title with all words capitalised at the beginning. Minor but I would go with Caps on all words (except and, or, etc.)

File: content/podcast/Cultural-Evolution-Implications-and-whats-next.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+date: 2023-09-10
+src: https://anchor.fm/s/????.m4a
+---
+
+In Episode 4 of the series Joseph Henrich and Rufus Pollock continue on the Culture Evolution conversation.
+
+In this second part of the conversation continuing on directly from episode 3, Rufus Pollock and Joe Henrich discuss the implications of cultural evolution for modern challenges. They explore the potential for intentional experimentation in creating cultural norms that promote trust, cooperation, group cohesion and a sense of community and belonging. 
+
+Rufus and Joe touch upon the idea that Western societies might be running on the fumes of values and norms that were cultivated by historical religious practices. They discuss the need to find ways to renew and revitalize these values, potentially by experimenting with intentional communities that incorporate elements of shared meaning, trust-building, ritual and cooperation. The talk moves on to discuss intentional communities and suggest that by allowing a variety of intentional communities to form and observing which ones thrive, societies could potentially find ways to address challenges and promote positive cultural evolution.
+
```
Comment (nathenf) [inline content/podcast/Cultural-Evolution-Implications-and-whats-next.md:13]: Add "This conversation forms part of the Cultural Evolution: A New Discipline is Born Series." and link to learn page

**Review by nathenf — COMMENTED** [review]
This all looks GREAT @laurenwigmore, I have made a few minor comments. You can make a note of them and then push it first then edit if you want, or figure out how to edit a PR 😄

**Comment by rufuspollock** [comment]
@laurenwigmore can we merge this? /cc @nathenf

**Comment by laurenwigmore** [comment]
> @laurenwigmore can we merge this? /cc @nathenf

Yes tomorrow (Thursday), I will, I need to make a few small amends and then everything Culturology will be live.

**Pull request merged**